### PR TITLE
🌐 [i18n-KO] Translated `package_reference/file_download.md` to Korean

### PR DESCRIPTION
**Pull request by seoyoung-3060**
# What does this PR do?

Translated the `package_reference/file_download.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/huggingface_hub/issues/1626

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

Team Hugging Face KREW, may you please review this PR? 
@harheem, @cjfghk5697, @seoulsky-field, @nuatmochoi, @boyunJang, @SeungAhSon, @seoyoung-3060, @junejae, @usr-bin-ksh, @fabxoe

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@jungnerd, @heuristicwave May you please review this PR?

### CODE REVIEW

**Review by seoulsky-field — COMMENTED** [review]
좋은 번역 감사합니다 :) glossary 참고하여 단어 하나만 수정하였습니다!

**Comment by cjfghk5697** [comment]
LGTM

File: docs/source/ko/package_reference/file_download.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+[[autodoc]]huggingface_hub.get_hf_file_metadata
+
+### HfFileMetadata[[huggingface_hub.HfFileMetadata]]
+
+[[autodoc]]huggingface_hub.HfFileMetadata
+
+## 캐싱[[caching]]
+
+위에 나열된 매소드들은 파일을 재다운로드하지 않도록 하는 캐싱 시스템과 함께 작동하도록 설계되었습니다. v0.8.0에서의 업데이트로, 캐싱 시스템은 Hub를 기반으로 하는 다양한 라이브러리 간의 공유 중앙 캐시 시스템으로 발전했습니다.
```
[the remark this answers is not in the document]
Comment (seoyoung-3060) [reply docs/source/ko/package_reference/file_download.md:33]: 좋은 의견 감사합니다! 다만, 딥러닝 용어사전 참고하여 "매소드"로 번역하였는데 확인 한번 부탁드립니다!

[the remark this answers is not in the document]
Comment (seoyoung-3060) [reply docs/source/ko/package_reference/file_download.md:33]: 아, 엑셀 glossary도 그렇고 TTA를 보니 "메소드"로 번역하는 방향이 맞는것같습니다! 수정하겠습니다 감사합니다!

**Review by Wauplin — APPROVED** [review]
Love it! :heart:

**Pull request merged**